ee/server/service/mdm_profiles.go

```diff
@@ -63,6 +63,18 @@ var fileVaultProfileTemplate = template.Must(template.New("").Option("missingkey
 			<key>PayloadVersion</key>
 			<integer>1</integer>
 		</dict>
+		<dict>
+            <key>dontAllowFDEDisable</key>
+            <true/>
+            <key>PayloadIdentifier</key>
+            <string>com.apple.MCX.62024f29-105E-497A-A724-1D5BA4D9E854</string>
```

Comment:
Note that this payload identifier was not added to the list of Fleet-managed payloads because it is not limited to FileVault settings. We should consider how we want to screen for changes to this on custom profiles.

Reply:
Does that mean that a `fleetctl apply` of custom settings would remove it?

Reply:
Gotcha, yeah I was a bit nervous seeing all those different PayloadIdentifiers in the same profile, but I think the one we care about (use for identification in custom profiles and all) is the one at the root of the xml, right? In `plist.dict.key[PayloadIdentifier]` in pseudo-xpath, I guess (haven't used that in years :D ).

Reply:
Yes, I think we're using the `ScreenPayloads` method, which checks if the `mobileconfig` uses one of the reserved PayloadIdentifiers (currently, "com.fleetdm.fleet.mdm.filevault" and "com.fleetdm.fleetd.config") or reserved PayloadTypes ("com.apple.security.FDERecoveryKeyEscrow", "com.apple.MCX.FileVault2", and "com.apple.security.FDERecoveryRedirect"). 

It would be nice to screen for the "com.apple.MCX" PayloadType, but that PayloadType can also be used for other [settings](https://developer.apple.com/documentation/devicemanagement/accounts). 

IIUC we are not screening for any of the unique PayloadIdentifiers that we set in the PayloadContent dict of our template (the ones that append the uuid to the payload type). I suppose we could, but I don't know if that would be worth the effort.

Reply:
Gotcha, thanks for the details!